src/Contract/API/Management/ClientsInterface.php

```diff
@@ -105,4 +105,67 @@ public function delete(
         string $id,
         ?RequestOptions $options = null,
     ): ResponseInterface;
+
+    /**
+     * Add new credentials to the specified Client.
+     *
+     * @param  string  $clientId Client to add credentials to
+     * @param  array<mixed>  $body  Additional body content to pass with the API request. See @see for supported options.
```

Comment:
I'm assuming these will follow this convention https://github.com/auth0/auth0-python/blob/master/auth0/management/client_credentials.py#L77

These links will just have to the top of the mngmt api docs while we wait for the docs to be published. (or you can leave these links out and update them when this is GA'd)

Reply:
Added in https://github.com/auth0/auth0-PHP/pull/700/commits/2eedc725954f386f41e17007b0097451075a1771, thanks!